CMakeLists.txt

```diff
@@ -1216,7 +1216,9 @@ if(NOT UNIX)
   if(CMAKE_INSTALL_SYSTEM_RUNTIME_LIBS)
     install(FILES ${CMAKE_INSTALL_SYSTEM_RUNTIME_LIBS} DESTINATION "bin")
     install(FILES ${CMAKE_INSTALL_SYSTEM_RUNTIME_LIBS} DESTINATION "Ruby" COMPONENT "RubyAPI")
-    install(FILES ${CMAKE_INSTALL_SYSTEM_RUNTIME_LIBS} DESTINATION "CSharp/openstudio/" COMPONENT "CSharpAPI")
+    if (BUILD_CSHARP_BINDINGS)
+      install(FILES ${CMAKE_INSTALL_SYSTEM_RUNTIME_LIBS} DESTINATION "CSharp/openstudio/" COMPONENT "CSharpAPI")
+    endif()
```

Comment:
Don't install the runtimes libs there if C# isn't selected to begin with.

This threw me off when I was looking at the CI-built packages, because actually the incremental runner does not build C#!